6-magic.md

```diff
@@ -101,53 +66,49 @@ In order to cast a spell, the caster must roll 2D6. If they equal or exceed the
-
-| D6 | Result | Difficulty | Type | Effect |
+| D6 | Result | Difficulty | Effect |
 | :-: | :-: | :-: | :-: |
-| 1 | The Hammer of Sigmar | 7 | Augment | This weapon of the faithful glows with a golden light, imbued as it is with the righteous power of Sigmar. | he wielder gains +2 Strength in hand-to-hand combat and all hits he inflicts cause double damage (e.g., 2 wounds instead of 1). The Priest must test each shooting phase he wants to use the Hammer. |
-| 2 | Hearts of Steel | 8 | Augment | Asaves of glory surround the servant of Sigmar, the faithful are heartened by the warrior god’s presence. Any allied warriors within 8" of the warrior become immune to Fear and All Alone tests. In addition, the whole warband gains +1 to any Rout tests they have to make. The effects of this spell last until the caster is knocked down, stunned or put out of action. If cast again the effects are not cumulative, i.e., the maximum bonus to Rout tests remains +1. |
-| 3 | Soulfire | 9 | Direct | Purifying flames surround the Priest and wipe out those who resist the righteous fury of the God-Emperor! All enemy models within 4" of the servant of Sigmar suffer a Strength 3 hit. No armor saves are allowed. The servants of darkness and Chaos are especially susceptible to Sigmar’s holy power. Undead and Possessed models in range suffer a Strength 5 hit instead. |
-| 4 | Shield of Faith | 6 | Augment | A shield of pure white light appears in front of the Priest. As long as his faith remains strong the shield will protect him. The Priest is immune to all spells. Roll at the beginning of each turn in the recovery phase. On a roll of 1 or 2 the shield disappears. |
-| 5 | Healing Hand | 5 | Augment | Laying hands upon a wounded comrade, the servant of Sigmar calls upon his Lord to heal the warrior’s wounds. Any one model within 2" of the Priest (including himself) may be healed. The warrior is restored to his full quota of Wounds. In addition, if any friendly models within 2" are stunned or knocked down, they immediately come to their senses, stand up, and continue fighting as normal. |
-| 6 | Armor of Righteousness | 9 | Augment | Impenetrable armor covers the Priest and the fiery image of a twin-tailed comet burns above his head. The Priest has an armor save of 2+ which replaces his normal armor save. In addition, he causes fear in his enemies and is therefore immune to fear himself. The power of the Armor of Righteousness lasts until the beginning of the Priest’s next Shooting phase. |
+| 1 | The Hammer of Sigmar | 7 | Augment. The caster's weapon is imbued with the white-hot light of Sigmar. Until the start of the caster's next magic phase they gain +1 Strength and any successful wounds (after resolving armour saves) are multipled into two wounds. |
```

Comment:
@Yther what made you decide not to have the spell type as a table column?

Reply:
Just a stylistic choice. The debug viewer only shows the preview of the previous update rather than updating constantly with my WIPs, meaning I couldn't see how the tables would look as I was writing them.  It is probably worth having the type as a column though, I can work on that today.

Reply:
It's actually very easy to change later so not a worry for now!